app/Http/Controllers/BarbersController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Barber;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Gate;
+
+class BarbersController extends Controller
+{
+    public function store(Request $request)
+    {
+        Barber::create($request->all());
+
+        return redirect()->route('barbershop.register');
+    }
+
+    public function show(Barber $barber)
+    {
+        return view('barber.profile')->with('barber', $barber);
```

Comment:
Often it helps to keep your view names and your method names in sync. It's easier to grok for new devs coming to the project and easier to digest; `BarberController@show` is a route named `barbers.show` which is served by a view at `resources\views\barbers\show.blade.php`.